src/components/Text/Text.tsx

```diff
@@ -33,28 +33,33 @@ interface TextProps
     inverted?: boolean;
     /**
      * Adjust color to indicate secondary information
+     * @deprecated
```

Comment:
Can we add a comment, explaining what prop to use instead? Otherwise the users will assume that there is no replacement for this property.

Reply:
Done